UPGRADING.md

```diff
@@ -34,15 +34,16 @@ We did our best to make this transition as painless as possible for you, so here
 #### That's great! What should I change in my code immediately after upgrading?
 
 * Add the corresponding adapter gem to your Gemfile (e.g. `faraday-net_http`). Ideally, this should replace
-  `faraday` altogether as these gems usually have Faraday already in their dependencies.
+  `faraday` altogether as these gems usually have Faraday already in their dependencies. Example Gemfile line:
+  ```ruby
+  gem 'faraday-net_http'
```

Comment:
When requiring, this should be `faraday/net_http`

Reply:
Unrelated: Right. I peeked in the Gemfile description, of the kwarg "require:" for `gem`, but it doesn't fully help the user.

https://bundler.io/v2.2/man/gemfile.5.html#REQUIRE-AS

I'll improve the documentation on "what to do in the app code".